code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -498,6 +498,11 @@ var/const/NO_EMAG_ACT = -50
 	access = list(access_distress, access_maint_tunnels, access_external_airlocks)
 	..()
 
+/obj/item/card/id/distress/fsf_ert
```

Comment:
If you set up a I'd card here then use it above aswell.

Also same path issue as above.